tests/integration_tests/data_steward/retraction/retract_deactivated_pids_test.py

```diff
@@ -1,66 +1,114 @@
+VALUES
+(1,'2009-07-25'),
+(2,'2009-03-14'),
+(3,'2009-11-18'),
+(4,'2009-11-25'),
+(5,'2009-09-20')
+""")
+
+TABLE_ROWS = {
+    'person':
+        JINJA_ENV.from_string("""
+INSERT INTO `{{table.project}}.{{table.dataset_id}}.{{table.table_id}}`
+(person_id, gender_concept_id, year_of_birth, birth_datetime, race_concept_id, ethnicity_concept_id)
+VALUES
+(1,8507,1989,'1989-07-25 01:00:00 UTC', 8527, 38003563),
+(2,8507,1975,'1975-03-14 02:00:00 UTC', 8527, 38003564),
+(3,8507,1981,'1981-11-18 05:00:00 UTC', 8527, 38003564),
+(4,8507,1991,'1991-11-25 08:00:00 UTC', 8527, 38003564),
+(5,8507,2001,'2001-09-20 11:00:00 UTC', 8527, 38003564)
+"""),
+    'observation':
+        JINJA_ENV.from_string("""
+INSERT INTO `{{table.project}}.{{table.dataset_id}}.{{table.table_id}}`
+(observation_id, person_id, observation_concept_id, observation_date, observation_datetime, observation_type_concept_id)
+VALUES
+(1001,1,0,'2008-07-25','2008-07-25 01:00:00 UTC',45905771),
+(1005,2,0,'2008-03-14','2008-03-14 02:00:00 UTC',45905771),
+(1002,3,0,'2009-11-18','2009-11-18 05:00:00 UTC',45905771),
+(1004,4,0,'2009-11-25','2009-11-25 08:00:00 UTC',45905771),
+(1003,5,0,'2010-09-20','2010-09-20 11:00:00 UTC',45905771)
+"""),
+    'death':
+        JINJA_ENV.from_string("""
+INSERT INTO `{{table.project}}.{{table.dataset_id}}.{{table.table_id}}`
+(person_id, death_date, death_datetime, death_type_concept_id)
+VALUES
+(2,'2008-03-12','2008-03-12 05:00:00 UTC',8),
+(3,'2011-01-18','2011-01-18 05:00:00 UTC',6)
+"""),
+    'drug_exposure':
+        JINJA_ENV.from_string("""
+INSERT INTO `{{table.project}}.{{table.dataset_id}}.{{table.table_id}}`
+(drug_exposure_id, person_id, drug_concept_id, drug_exposure_start_date, drug_exposure_start_datetime,
+drug_exposure_end_date, drug_exposure_end_datetime, verbatim_end_date, drug_type_concept_id)
+VALUES
+(2002,1,50,'2008-06-05','2008-06-05 01:00:00 UTC','2010-07-05','2008-06-05 01:00:00 UTC','2011-04-11',87),
+(2003,2,21,'2008-11-22','2008-11-22 02:00:00 UTC',null,null,'2010-06-18',51),
+(2004,3,5241,'2009-08-03','2009-08-03 05:00:00 UTC',null,null,'2009-12-26',2754),
+(2005,4,76536,'2010-02-17','2010-02-17 08:00:00 UTC',null,null,'2008-03-04',24),
+(2006,5,274,'2009-04-19','2009-04-19 11:00:00 UTC',null,'2010-11-19 01:00:00 UTC','2011-10-22',436)
+""")
+}
+
+MAPPING_TABLE_ROWS = {
+    '_mapping_observation':
+        JINJA_ENV.from_string("""
+INSERT INTO `{{table.project}}.{{table.dataset_id}}.{{table.table_id}}`
+(observation_id, src_hpo_id)
+VALUES
+(1001,'hpo_1'),
+(1002,'hpo_2'),
+(1003,'hpo_3'),
+(1004,'PPI/PM'),
+(1005,'hpo_4'),
+(1006,'hpo_4')
+""")
+}
+
+EXT_TABLE_ROWS = {
+    'drug_exposure_ext':
+        JINJA_ENV.from_string("""
+INSERT INTO `{{table.project}}.{{table.dataset_id}}.{{table.table_id}}`
+(drug_exposure_id, src_id)
+VALUES
+(2002,'PPI/PM'),
+(2003,'EHR Site 50'),
+(2004,'EHR Site 22'),
+(2005,'EHR Site 9'),
+(2006,'EHR Site 17')
+""")
+}
 
 
 class RetractDeactivatedEHRDataBqTest(unittest.TestCase):
```

Comment:
Thanks for pointing this out. It would be ideal to have a retraction class to test deactivation and retraction scripts. We could add this while refactoring retraction.